yb-voyager/cmd/exportSchema.go

```diff
@@ -59,8 +60,21 @@ var exportSchemaCmd = &cobra.Command{
 	},
 
 	Run: func(cmd *cobra.Command, args []string) {
+		err := runAssessMigrationCmdBeforExportSchemaIfRequired(cmd)
```

Comment:
I think this might be too early in the process. 
`exportSchema` has a lot more checks:
1. start-clean check with prompt
2. source db connection check 
3. general guardrails. 

Should we call `runAssessMigrationCmdBeforExportSchemaIfRequired` after all that?